recipes/belt/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "BELT" %}
+{% set version = "BELTv1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qianglbl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: cbfe5081595ba91bb88e4cb9deec001f60c73571a8cc10788ac60d789089f843
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  script: 
+    - ln -sf $BUILD_PREFIX/bin/$CONDA_TOOLCHAIN_HOST-gfortran $BUILD_PREFIX/bin/gfortran
+    - cd src
+    - make -f Makefile
+    - mkdir -p $PREFIX/bin
+    - cp xeblt $PREFIX/bin
+    - cp -r ../examples/example1 $PREFIX/test
```

Comment:
Please don't put example data into the package.

Reply:
The test has been removed.